engine/runtime/src/test/scala/org/enso/compiler/test/pass/resolve/ExpressionAnnotationsTest.scala

```diff
@@ -38,7 +43,14 @@ class AnnotationsTest extends CompilerTest {
       * @return [[ir]], with tail call analysis metadata attached
       */
     def analyse(implicit context: ModuleContext): IR.Module = {
-      Annotations.runModule(ir, context)
+      ExpressionAnnotations.runModule(ir, context)
+    }
+  }
+
+  // TODO [AA] Doc this
```

Comment:
.

Reply:
but also, don't `doc this` – a private helper in a test with one-line implementation copy-pasted across all compiler tests. Documenting things like this is one reason why these tests are so horrible to write